app/account/schema.ts

```diff
@@ -0,0 +1,22 @@
+import { zodResolver } from '@hookform/resolvers/zod'
+import * as z from 'zod'
+
+const accountSchema = z
+  .object({
+    email: z.string().email('This is not valid email address'),
+    name: z.string().min(1),
```

Comment:
Could you add a message like trip form? The message is not user friendly. 
<img width="543" alt="Screenshot 2024-03-02 at 2 36 33 PM" src="https://github.com/tabi-memo/frontend/assets/66394413/d4b8c50c-7e41-49ac-976a-05bc4a014624">